source/fundamentals/crud/read-operations/distinct.txt

```diff
@@ -0,0 +1,252 @@
+            :language: csharp
+            :dedent:
+
+         .. output::
+            :visible: false
+
+            Bronx
+            Brooklyn
+            Manhattan
+            Missing
+            Queens
+            Staten Island
+
+The operation returns a cursor that you can iterate through to access each distinct ``borough``
+field value. Although several documents have the same value in the ``borough`` field, each value appears
+in the results only once.
+
+Retrieve Values Across Specified Documents
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can provide a **query filter** to the ``Distinct()`` and ``DistinctAsync()`` methods
+to find distinct field values within a subset of documents in a collection. A query filter
+is an expression that specifies search criteria used to match documents in an operation.
+For more information about creating a query filter, see the :ref:`csharp-specify-query` guide.
+
+The following example retrieves the distinct values of the ``borough`` field for
+all documents that have a ``cuisine`` field value of ``"Italian"``. Select the
+:guilabel:`Asynchronous` or :guilabel:`Synchronous` tab to see the corresponding code.
+
+.. tabs::
+    
+   .. tab:: Asynchronous
+      :tabid: distinct-async
+
+      .. io-code-block::
+         :copyable:
+
+         .. input:: /includes/fundamentals/code-examples/Distinct.cs
+            :start-after: start-distinct-with-query-async
+            :end-before: end-distinct-with-query-async
+            :language: csharp
+            :dedent:
+
+         .. output::
+            :visible: false
+
+            Bronx
+            Brooklyn
+            Manhattan
+            Queens
+            Staten Island
+
+   .. tab:: Synchronous
+      :tabid: distinct-sync
+
+      .. io-code-block::
+         :copyable:
+
+         .. input:: /includes/fundamentals/code-examples/Distinct.cs
+            :start-after: start-distinct-with-query
+            :end-before: end-distinct-with-query
+            :language: csharp
+            :dedent:
+
+         .. output::
+            :visible: false
+
+            Bronx
+            Brooklyn
+            Manhattan
+            Queens
+            Staten Island
+
+Modify Distinct Behavior
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can modify the behavior of the ``Distinct()`` and ``DistinctAsync()`` methods by
+providing a ``DistinctOptions`` instance as an optional parameter. The following table
+describes the properties you can set on a ``DistinctOptions`` instance:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Method
+     - Description
+
+   * - ``Collation`` 
+     - | Sets the collation to use for the operation.
+       | **Parameter Type**: ``Collation``
+
+   * - ``MaxTime``
+     - | Sets the maximum amount of time that the operation can run.
+       | **Parameter Type**: ``TimeSpan``
+
+   * - ``Comment`` 
+     - | Attaches a comment to the operation.
+       | **Parameter Type**: ``BsonValue`` or ``String``
```

Comment:
Other pages in the C# docs use "Data type" instead of "Parameter type". Probably good to be consistent.

For non-primitive types, I also like to link to an explanation of the type, like [here](https://github.com/mongodb/docs-csharp/pull/289/files#diff-fb204313ea3602e39b8d8e1837017052fd2f07c92c0e7c0b4da15e09989f26acR179)